openfe/setup/atom_mapping/perses_scorers.py

```diff
@@ -0,0 +1,117 @@
+# This code is part of OpenFE and is licensed under the MIT license.
+# For details, see https://github.com/OpenFreeEnergy/openfe
+
+from typing import Callable
+
+from openfe.utils import requires_package
+
+try:
+    from perses.rjmc.atom_mapping import AtomMapper, AtomMapping
+except ImportError:
```

Comment:
As per mapper we should block the import and make sure users are suitably errored out.